src/routes/safe/container/index.tsx

```diff
@@ -67,7 +70,17 @@ const Container = (): React.ReactElement => {
           path={`${matchSafeWithAddress?.path}/transactions`}
           render={() => wrapInSuspense(<TxsTable />, null)}
         />
-        <Route exact path={`${matchSafeWithAddress?.path}/apps`} render={() => wrapInSuspense(<Apps />, null)} />
+        <Route
```

Comment:
I would remove the <Route in case safe-apps are disabled.
`{safeAppsEnabled && <Route...`